assets/css/app.scss

```diff
@@ -1,4 +1,5 @@
 // TailwindCSS
+@import 'tailwindcss/base';
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
```

Comment:
You actually don't need. You can include the mbta metro stylesheet that has these as well as the mapgl styles.

https://github.com/anthonyshull/mbta_metro?tab=readme-ov-file#install-mbta-metro

Reply:
Since this PR is pretty old I'll tackle including the mbta metro stylesheet in a later PR. At least this PR gets us one step closer to Tailwind-style-compatibility. 👍🏼